### PR TITLE
feature(welcome): adding a one-time welcome modal

### PR DESCRIPTION
**Pull request by andrerfneves**
feature(welcome): adding a one-time welcome modal

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -20,17 +20,29 @@ type State = {
...
+};
+
 export class App extends Component<Props, State> {
   state = {
     themeMode: getInitialTheme(),
   };
 
   componentDidMount() {
     electronStore.set(THEME_MODE, getInitialTheme());
+    electronStore.set(THEME_MODE, getWelcomeModal());
```
Comment (andrerfneves) [inline app/app.js]: Incorrect

File: app/components/modal.js [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import React, { PureComponent, Fragment, type Element } from 'react';
 import { createPortal } from 'react-dom';
 import styled from 'styled-components';
+import { tsImportEqualsDeclaration } from '@babel/types';
```
Comment (andrerfneves) [inline app/components/modal.js]: Dsiregard. Was an automatic import from VSCode, will be removed

**Comment by andrerfneves** [comment]
# Tasks:
* [ ] Only trigger welcome modal when first time opening app (DISPLAY_WELCOME_MODAL inside Electron store is `true`)
* [ ] Solve issue with Modal not closing on hitting Submit/onConfirm button
* [ ] Write the text inside of the Welcome Modal
* [ ] Move `DISPLAY_WELCOME_MODAL` to be a constant variable instead of a string

**Comment by andrerfneves** [comment]
@georgelima

**Comment by georgelima** [comment]
fixed @andrerfneves

**Pull request merged**